### ISSUE TITLE
HDFS2FileSink should support the Hadoop Load Balancing.

### ISSUE THREAD

**Issue by NMmayoedu** [IBMStreams/streamsx.hdfs#114]
This issue originated from dual/parallel writes to Hadoop HDFS store using HDFS2FileSink Operator on 2 HA cluster nodes with Laod Balancing. (Cloudera)
I am able to connect via the current HDFS2FileSink operator using  “configPath” parameter successfully – Yes (but still the data is not written – I beleive it’s the Redirect URL issue)
However to achieve this, I had to hardcode the fs.defaultFS property in core.xml to point to current active node from our cluster.
<property>
              <name>fs.defaultFS</name>
              <value>hdfs://XXXXXXXXXXXXX</value>
           
    </property>
    
This is a possibly an issue because every couple of weeks our active/passive node changes on Hadoop cluster.
Ideally we would expect the HDFS2FileSink operator to handle the URL redirects from our Load Balancing URL.

Earlier we ran into similar redirection issues while working through HTTPPost operator.
This issue was resolved in the new inet toolkit with availability of HTTPRequest operator.

I believe the HDFS2FileSink operator is also failing due to redirect issues when going against Load Balancer.

Some resources which cover the redirection topics:
https://hadoop.apache.org/docs/r2.6.4/hadoop-project-dist/hadoop-hdfs/WebHDFS.html

**Comment by anouri** [comment]
The problem was not with HDFS2FileSink or  Load Balancing.
For an HA HDFS server it is required to configure the NameNodes in the hdfs-core.xml file.
The problem is already solved.

**Comment by NMmayoedu** [comment]
The problem is solved yes, but your explanation confuses me. 
"For an HA HDFS server it is required to configure the NameNodes in the hdfs-core.xml file"
That is exactly how we have configured them since the time we have been using HDFS2FileSink version 4.4.1.
You were well aware of our HA configurations and I am still using the same set of core-site.xml and hdfs-site.xml configurations as nothing has changed on those configurations on our side for last 3 Years.

You have been privy to our TESTS and configurations on couple of occasions through WebEx calls.
Even now that it is working with the "configPath" parameter in the newly released HDFS2FileSink version 5.1.0, the only change that I have made to my SPL project is to swap out the Toolkit versions.
Also all my other configuration files are still the same including our Hadoop Lib 2.7.3.2.5.3.0-37.

The only variable in this complete setup was to switch to newer HDFS2FileSink version 5.1.0 to make it work, so it escapes me when you mention about hdfs-core.xml which has never changed.
